execute/exectypes/outcome.go

```diff
@@ -7,8 +7,40 @@ import (
 	cciptypes "github.com/smartcontractkit/chainlink-common/pkg/types/ccipocr3"
 )
 
+type PluginState int
+
+const (
+	// Unknown is the default value indicating there was no previous state.
+	Unknown PluginState = iota
+	GetCommitReports
+	GetMessages
+	Filter
```

Comment:
First state is self explanatory but these might need some comments, especially `Filter`

Reply:
Or maybe call it `FilterValidMessages`?

Reply:
Added comments :+1: